docs/docs/kubernetes/operator/installation/helm.md

```diff
@@ -0,0 +1,92 @@
+
+## Example - Chart repository
+
+This will install the operator in the `trivy-system` namespace and configure it to scan all namespaces, except `kube-system` and `trivy-system`:
+
+```bash
+helm repo add aqua https://aquasecurity.github.io/helm-charts/
+helm repo update
+helm install trivy-operator aqua/trivy-operator \
+   --namespace trivy-system \
+   --create-namespace \
+   --set="trivy.ignoreUnfixed=true" \
+   --version {{ var.chart_version }}
+```
+
+## Example - Download the chart
+
+This will install the operator in the `trivy-system` namespace and configure it to scan all namespaces, except `kube-system` and `trivy-system`:
+
+```bash
+git clone --depth 1 --branch {{ git.tag }} https://github.com/aquasecurity/trivy-operator.git
+cd trivy-operator
+helm install trivy-operator ./deploy/helm \
+--namespace trivy-system \
+--create-namespace \
+--set="trivy.ignoreUnfixed=true"
+```
+
+## Post install sanity check
+
+Check that the `trivy-operator` Helm release is created in the `trivy-system` namespace, and it has status `deployed`:
+
+```console
+$ helm list -n trivy-system
+NAME              	NAMESPACE         	REVISION	UPDATED                             	STATUS  	CHART                   	APP VERSION
+trivy-operator	trivy-system	1       	2021-01-27 20:09:53.158961 +0100 CET	deployed	trivy-operator-{{ var.chart_version }}	{{ git.tag[1:] }}
+```
+
+To confirm that the operator is running, check that the `trivy-operator` Deployment in the `trivy-system`
+namespace is available and all its containers are ready:
+
+```console
+$ kubectl get deployment -n trivy-system
+NAME                 READY   UP-TO-DATE   AVAILABLE   AGE
+trivy-operator   1/1     1            1           11m
+```
+
+If for some reason it's not ready yet, check the logs of the Deployment for errors:
+
+```
+kubectl logs deployment/trivy-operator -n trivy-system
+```
+
+## Advanced Configuration
+
+The Helm chart supports all available [installation modes](./../configuration.md#install-modes) of Trivy Operator. 
+
+Please refer to the chart's [values] file for configuration options.
+
+## Uninstall
+
+You can uninstall the operator with the following command:
+
+```
+helm uninstall trivy-operator -n trivy-system
+```
+
+You have to manually delete custom resource definitions created by the `helm install` command:
+
+!!! danger
+    Deleting custom resource definitions will also delete all security reports generated by the operator.
+
+    ```
+    kubectl delete crd vulnerabilityreports.aquasecurity.github.io
+    kubectl delete crd clustervulnerabilityreports.aquasecurity.github.io
+    kubectl delete crd configauditreports.aquasecurity.github.io
+    kubectl delete crd ciskubebenchreports.aquasecurity.github.io
+    kubectl delete crd kubehunterreports.aquasecurity.github.io
```

Comment:
remove the kube hunter line : `kubectl delete crd kubehunterreports.aquasecurity.github.io`

Reply:
should the one above it also be removed?